### ISSUE TITLE
Checking if account verification email has clickable URL

### ISSUE THREAD

**Issue by yannisbarlas** [Coko-Foundation/hhmi#407]
> **Migrated from GitLab**
> **Original Author:** Christopher Vargas (`@vargasc`)
> **Original Link:** https://gitlab.coko.foundation/coko-org/products/hhmi/hhmi/-/issues/69

The hyperlink in the email provided for account verification wasn’t recognized as a hyperlink to open a browser/new webpage by alt nav technology (dragon naturally speaking) and Voiceover

I believe this may have been corrected in the current version of the account verification email, but I tried to look back through tickets and did not see anything specifically covering this.

**Comment by yannisbarlas** [comment]
> **Migrated Comment**
> **Author:** Yannis Barlas (`@yannis`)
> **Date:** 2022-11-01T14:38:22.149Z

It's a normal html anchor tag, so it should be accessible. That's for the actual link. I'm also including the link as plain text underneath because there was an issue with some mail clients not reading the link. Maybe the issue refers to that?

**Comment by yannisbarlas** [comment]
> **Migrated Comment**
> **Author:** Dione Mentis (`@DioneMentis`)
> **Date:** 2023-04-11T05:27:00.881Z

Hi `@conneelyb` , yes our team has completed all accessibility feedback
